tests/helm/validate_charts.sh

```diff
@@ -61,6 +61,7 @@ check_helm_installed() {
 validate_s3_endpoint_required() {
   local chart_dir="$CHARTS_DIR/scality-mountpoint-s3-csi-driver"
   local node_yaml="$chart_dir/templates/node.yaml"
+  local values_yaml="$chart_dir/values.yaml"
```

Comment:
Probably we do not need such test as we are have a sample value, but still making the needed change.